src/cmake/externalpackages.cmake

```diff
@@ -238,6 +238,21 @@ if (USE_CUDA OR USE_OPTIX)
     if (NOT ${nvptx_index} GREATER -1)
         message (FATAL_ERROR "NVTPX target is not available in the provided LLVM build")
     endif()
+
+    if (${CUDA_VERSION_MAJOR} GREATER 8 AND ${LLVM_VERSION} LESS 6)
```

Comment:
FYI: we've had a little bit of trouble with llvm6 at our site, so we're hoping to stick with llvm 5.0.1 for now. We are currently on Cuda 8.0 and Optix 5.0. A hard requirement for either LLVM6 or Cuda 9 would force up to need some upgrades (including drivers across the facility, which is a bigger PITA, I think we are on a driver version that's slightly too old for Cuda 9).